packages/quantic/force-app/main/default/lwc/quanticUserActionsToggle/e2e/pageObject.ts

```diff
@@ -0,0 +1,71 @@
+import {Locator, Page, Request, Response} from '@playwright/test';
+import {isUaCustomEvent} from '../../../../../../playwright/utils/requests';
+
+const userActionsRequestRegex =
+  /\/rest\/organizations\/.*\/machinelearning\/user\/actions$/;
+
+export class UserActionsToggleObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get userActionsToggleButton(): Locator {
+    return this.page.getByRole('button', {name: /User actions/i});
+  }
+
+  get userActionsModal(): Locator {
+    return this.page.locator(
+      'c-quantic-user-actions-toggle > c-quantic-modal > div'
+    );
+  }
+
+  get userActionsModalCloseButton(): Locator {
+    return this.page.locator(
+      'c-quantic-user-actions-toggle > c-quantic-modal button[title="Close"]'
```

Comment:
`this.page.getByRole('button', {name: /Close/i});` ? Why no get by role with this one?